app/services/search/school_search.rb

```diff
@@ -36,12 +37,12 @@ def total_count
   end
 
   def total_filters
-    filter_counts = %i[education_phase key_stage special_school job_availability].map { |filter| search_criteria[filter]&.count || 0 }
+    filter_counts = %i[education_phase key_stage special_school job_availability organisation_types].map { |filter| search_criteria[filter]&.count || 0 }
     filter_counts.sum
   end
 
   def clear_filters_params
-    active_criteria.merge({ education_phase: [], key_stage: [], special_school: [], job_availability: [] })
+    active_criteria.merge({ education_phase: [], key_stage: [], special_school: [], job_availability: [], organisation_type: [] })
```

Comment:
Shouldn't this be plural as the filter is named `organisation_types` ?

Reply:
Yep! Thanks Marc, good spot!